portal-ui/src/screens/Console/Buckets/ViewBucket/PolicySelectors.tsx

```diff
@@ -0,0 +1,216 @@
+// This file is part of MinIO Kubernetes Cloud
+// Copyright (c) 2021 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { useCallback, useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { createStyles, Theme, withStyles } from "@material-ui/core/styles";
+import { LinearProgress } from "@material-ui/core";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import SearchIcon from "@material-ui/icons/Search";
+import TextField from "@material-ui/core/TextField";
+import { policySort } from "../../../utils/sortFunctions";
+import {
+  actionsTray,
+  selectorsCommon,
+} from "../Common/FormComponents/common/styleLibrary";
+import { setModalErrorSnackMessage } from "../../../actions";
+import api from "../../../common/api";
+import TableWrapper from "../Common/TableWrapper/TableWrapper";
+import {Policy} from "../Policies/types";
+
+interface ISelectPolicyProps {
+  classes: any;
+  selectedPolicy?: string;
+  setSelectedPolicy: any;
+  setModalErrorSnackMessage: typeof setModalErrorSnackMessage;
+}
+
+interface PolicyList {
+    policies: Policy[];
+    total: number;
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    seeMore: {
+      marginTop: theme.spacing(3),
+    },
+    paper: {
+      display: "flex",
+      overflow: "auto",
+      flexDirection: "column",
+      paddingTop: 15,
+      boxShadow: "none",
+    },
+    addSideBar: {
+      width: "320px",
```

Comment:
When css properties are pixels in JS, we try to just use the number, if it is a composed property or the unit is not pixels, we apply the string with the unit
```suggestion
      width: 320,
```